src/SWP/Bundle/CoreBundle/Resources/config/doctrine-orm/FacebookInstantArticlesFeed.orm.yml

```diff
@@ -0,0 +1,45 @@
+SWP\Bundle\CoreBundle\Model\FacebookInstantArticlesFeed:
+    type: entity
+    table: swp_fbia_feed
+    id:
+        id:
+            type: integer
+            id: true
+            generator:
+                strategy: AUTO
+    fields:
+        tenantCode:
+            type: string
+        mode:
```

Comment:
same in this case, this mapping for facebook specific field should go to FBIABundle